st_package_reviewer/check/file/check_messages.py

```diff
@@ -27,10 +38,11 @@ def check(self):
                     self.fail("unable to load `messages.json`", exception=e)
                     return
 
+            prefix_regex = '^(' + '|'.join(list(self.prefixes)) + ')'
```

Comment:
If you want to be *really* correct in a probably meaningless way, you can add `re.escape()`. Something like

```suggestion
            prefix_regex = '^(' + '|'.join(re.escape(p) for p in self.prefixes) + ')'
```

But I doubt that it affects anyone currently or will in the future.